code/datums/supplypacks.dm

```diff
@@ -528,7 +528,7 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = access_armory
 	group = "Security"
 
-/datum/supply_packs/security/armory/knight //why seperate them
+/datum/supply_packs/security/armory/knight

```

Comment:
@PJB3005 yeah, you caught me, i'm a shit coder and i have no idea how to fix any of this
might be best to just revert anything i added